statina/main.py

```diff
@@ -6,11 +6,9 @@
 from statina.exeptions import CredentialsError
 
 import statina.API.external.api.api_v1.endpoints as external_api_v1
-import statina.API.internal.api.api_v1.endpoints as internal_api_v1
 import statina.API.v2.endpoints as external_api_v2
 
 external_versions = {"v1": external_api_v1}
-internal_versions = {"v1": internal_api_v1}
 
 
 def external(version: str) -> FastAPI:
```

Comment:
no need for this function anymotre. Also shouldn't be called external_app but just app